Modix.Data/Models/BehaviourConfiguration.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Modix.Data.Models
+{
+    public class BehaviourConfiguration
+    {
+        [Key]
```

Comment:
For consistency, recommend adding `Required` and `DatabaseGenerated(DatabaseGeneratedOption.Identity)`

Reply:
We can deal with this in #94.